ansible/fhir-stress-test.yml

```diff
@@ -39,142 +23,57 @@
-          - "{{ linux_to_ad_via_ssd_group }}"
-          - "{{ temp_app_pub_group }}"
-        instance_type: "{{ ec2_instance_type }}"
-        image: "{{ ami_id_rhel }}"
-        region: "{{ region }}"
-        vpc_subnet_id: "{{ subnet_id }}" 
-        wait: true
-        exact_count: 1
-        count_tag:
-          Name: bluebutton-stress-client
-          Application: "{{ app_tag }}"
-        instance_tags:
-          Name: bluebutton-stress-client
-          APPLICATION: "{{ app_tag }}"
-          TIER: "{{ tier_tag }}"
-          PURPOSE: bluebutton-stress-client
-          ENVIRONMENT: "{{ env_tag }}"
-          CreatedBy: "{{ whoami.stdout }}"
-      register: ec2_client
-    
-    - name: EC2 - Provision 'bluebutton-stress' servers
-      ec2:
-        aws_access_key: "{{ session_credentials.sts_creds.access_key }}"
-        aws_secret_key: "{{ session_credentials.sts_creds.secret_key }}"
-        security_token: "{{ session_credentials.sts_creds.session_token }}"
-        key_name: "{{ ec2_key_name }}"
-        group_id:
-          - "{{ gss_common_services }}"
-          - "{{ app_server_security_group }}"
-          - "{{ general_services_group }}"
-          - "{{ linux_to_ad_via_ssd_group }}"
-          - "{{ temp_app_pub_group }}"
-        instance_type: "{{ ec2_instance_type }}"
-        image: "{{ ami_id_rhel }}"
-        region: "{{ region }}"
-        vpc_subnet_id: "{{ subnet_id }}" 
-        wait: true
-        # The default of 300s here wasn't cutting it for 8 servers.
-        wait_timeout: 600
-        exact_count: "{{ num_servers }}" 
-        count_tag:
-          Name: bluebutton-stress-server
-          Application: "{{ app_tag }}"
-        instance_tags:
-          Name: bluebutton-stress-server
-          APPLICATION: "{{ app_tag }}"
-          TIER: "{{ tier_tag }}"
-          PURPOSE: bluebutton-stress-server
-          ENVIRONMENT: "{{ env_tag }}"
-          CreatedBy: "{{ whoami.stdout }}"
-      register: ec2_servers
-    
-    - name: Create inventory group 'client'
-      add_host:
-        hostname: "{{ item.private_dns_name }}"
-        groupname: client
-      with_items: "{{ ec2_client.instances }}"
-
-    - name: Create inventory group 'servers'
-      add_host:
-        hostname: "{{ item.private_dns_name }}"
-        groupname: servers
-      with_items: "{{ ec2_servers.instances }}"
-
-    - name: Wait for SSH to come up
-      wait_for:
-        host: "{{ item.private_dns_name }}"
-        port: 22
-        timeout: 320
-        state: started
-      with_flattened:
-        - "{{ ec2_client.instances }}"
-        - "{{ ec2_servers.instances }}"
+- name: Configure Systems for Ansible Pipelining
+  hosts: client:servers
+  vars:
+    ansible_ssh_pipelining: false
+  roles:
+    - rhel_ansible_pipelining
 
 - name: Configure roles common to all servers 
   hosts: client:servers
   become: True
   gather_facts: True
-  environment: "{{proxy_env}}"
+#  environment: "{{proxy_env}}"
```

Comment:
Do these still need to be commented out (now that you changed the default in the JMeter JMX file)? If not, uncomment them.